packages/app-builder-lib/src/electron/ElectronFramework.ts

```diff
@@ -181,6 +181,18 @@ async function unpack(prepareOptions: PrepareApplicationStageDirectoryOptions, o
       return
     }
     await executeAppBuilder(["unpack-electron", "--configuration", JSON.stringify([options]), "--output", appOutDir, "--distMacOsAppName", distMacOsAppName])
+    if (packager.config.downloadAlternateFFmpeg) {
+      log.info("downloading and unpacking alternate FFmpeg")
+      const ffmpegOptions = Object.assign({}, options)
+      ffmpegOptions.customFilename = `ffmpeg-v${options.version}-${options.platform}-${options.arch}.zip`
+      let ffmpegLocation = path.join(appOutDir, "..")
```

Comment:
Would `path.dirname(appOutDir)` work better here? Either that or `path.resolve`?

Reply:
yeah, probably best to just stick to `path.dirname(appOutDir)` to avoid the `..`. I'll fix.